go/vt/vttablet/tabletserver/schema/load_table.go

```diff
@@ -157,3 +171,39 @@ func getNum(in map[string]string, key string) (int, error) {
 	}
 	return v, nil
 }
+
+func getCols(in map[string]string, key string) ([]string, error) {
+	sv := in[key]
+	if sv == "" {
+		return nil, fmt.Errorf("attribute %s not specified for message table", key)
+	}
+	cols := strings.Split(sv, ";")
```

Comment:
personal preference but after so many years my brain sees the semi-colon in SQL as a hard stmt/context delimiter. I can see that comma has its own problems in this context, but I think I would prefer pipe as it's not uncommon to use that as a column separator in output.

Another thing I don't feel super strongly about and leave up to you to make the final call on.

Reply:
done